src/pheval/utils/utils.py

```diff
@@ -0,0 +1,143 @@
+"""Contains all pheval utility methods"""
+import logging
+import random
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+
+info_log = logging.getLogger("info")
+
+
+def rand(self: pd.DataFrame, min_num: int, max_num: int, scramble_factor: float) -> float:
```

Comment:
is "self" a good variable name? it seems this is used to denote methods that belong to a class?